cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -257,16 +336,27 @@ with the metadata tags, as a list of strings, passed in the JSON request body::
    * - ``404 NOT FOUND``
      - The entity or program for which tags are being set was not found
 
+**Note**: Tags can be added to the metadata of the specified entity only in *user* scope using
+this API.
+
```

Comment:
**Note**: When using this API, tags can be added to the metadata of the specified entity only in the *user* scope.